### DISCUSSION TITLE
Events can't be longer than 365 days.

### DISCUSSION THREAD

**Discussion by dowadidi** [solspace/craft-calendar#25] [Questions & Answers] [answered]
Is there a particular reason why an event can't be longer than 365 days? Is it possible to add an option to skip this validation?

https://github.com/solspace/craft3-calendar/blob/master/src/Elements/Event.php#L36

**Comment by kjmartens** [comment, marked as answer]
Hi @dowadidi, this is to help prevent performance issues with Calendar. We've found that events that span longer periods of time are more likely to run into issues with performance or not displaying correctly later on. I've made a note about this, however, for future consideration to perhaps make this a setting that can be disabled at your own risk. :)
